samcli/commands/deploy/guided_context.py

```diff
@@ -114,6 +117,18 @@ def guided_prompts(self, parameter_override_keys):
         self.save_to_config = save_to_config
         self.confirm_changeset = confirm_changeset
 
+    def prompt_authorization(self, parameter_overrides):
+        auth_required_per_resource = auth_per_resource(parameter_overrides, get_template_data(self.template_file))
+
+        for resource, authorization_required in auth_required_per_resource:
+            if not authorization_required:
+                auth_confirm = confirm(
+                    f"\t{self.start_bold}{resource} may not have authorization defined, Is this okay?{self.end_bold}",
```

Comment:
if a user have samconfig.toml file with guided deploy configured and is being used in CI/CD, will this prompt break them?

Reply:
we dont save this value into samconfig

Reply:
no, its only on guided.